.gitignore

```diff
@@ -0,0 +1,13 @@
+_ss_environment.php
+.buildpath
+.project
+.settings
+.solr
+.ea
+.idea
+.DS_Store
+.env
+/assets
+/silverstripe-cache
+/resources
+/vendor
```

Comment:
I don't think we need this file really

Reply:
It's provided by most other recipes. I think it provides some necessary value, particularly if used in a `create-project` manner.

Reply:
Yeah I'm not sure - we don't have one in the recipe-core or recipe-cms SilverStripe recipes either. Perhaps they all need one, but that seems like duplication. If anything we'd add it to silverstripe/recipe-core as a project file so all recipes can use it

Reply:
Yeah, we do :>
https://github.com/silverstripe/recipe-core/blob/1.0/.gitignore
https://github.com/silverstripe/recipe-cms/blob/1.0/.gitignore

The difference between these and the one I'm trying to provide here is that they core recipes don't cover the one dependency we have for Blog that still sits in the site root (next line done from this comment; `/gridfield-bulk-editing-tools`)

I suppose we could just submit a PR.

Also `/assets` appears to be missing from the core ones.

Reply:
Ah right, sorry - I misunderstood this. Yep OK it's fine, but rather than add GFBET to .gitignore lets keep them the same as core (maybe even add a comment at the top like `# See silverstripe/recipe-core` or something, and wait for GFBET to become a vendor module

Reply:
Do you think it could be possible to merge for now (as this is targeting a `beta` release), and we open another issue to follow up on that dependency?

Reply:
Yep, good idea

Reply:
It's pretty safe to have this file. Maybe we don't need so much extra stuff in it (e.g. `.DS_Store`).